src/06oct25/main.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include "merge_sort.h"
+
+int main(void)
+{
+    int number;
+    int data[100];
+    int original_data[100];
```

Comment:
Переменные вот прям всегда нужно инициализировать. И стилю следовать.
```suggestion
    int number = 0;
    int data[100] = { };
    int originalData[100] = { };
```